demos/text_detection_demo/src/text_recognition.cpp

```diff
@@ -25,6 +25,29 @@ namespace  {
         }
         *prob = 1.0f / static_cast<float>(sum);
     }
+
+    void softmax(const std::vector<float>::const_iterator& begin, const std::vector<float>::const_iterator& end, std::vector<float> &prob)
+    {
+        std::transform(begin, end, prob.begin(), static_cast<double(*)(double)>(std::exp));
+        float sum = std::accumulate(prob.begin(), prob.end(), 0.0f);
+        for (int i = 0; i < static_cast<int>(prob.size()); i++)
+            prob[i] /= sum;
+    }
+
+    struct BeamElement
```

Comment:
Could you move `{ ` char to the same line as the struct declaration?
(Everywhere in the demo this style is used.)